.gitpod.yml

```diff
@@ -16,6 +16,7 @@ tasks:
       node ./webstone/packages/create-webstone-app/bin webstone-dev-app
     command: |
       cd webstone-dev-app
+      while [ ! -f ../webstone/packages/cli/dist/bin.js ]; do sleep 1; done
```

Comment:
The other terminal takes a moment to run `pnpm dev` for all packages. This new while-loop ensures we wait until the CLI is ready in dev mode.